### ISSUE TITLE
.ignore doesn't understand leading / on directory name properly (for .gitignore)

### ISSUE THREAD

**Issue by adobeDan** [hsz/idea-gitignore#340]
My .gitignore file has this line:
```
/build/
```
which should only match a top-level build directory, and my project has this directory at the root level:
```
misc/build/
```
Using the .ignore plugin, I get this message:
```
 Ignored files found that are tracked: .ignore plugin found some files that are tracked with Git which are also listed in .gitignore file
```
about all the files in `misc/build/` and its subdirectories.  But `git check-ignore` and `git status --ignored` correctly show all the files in in misc/build as _not_ ignored.  So apparently the .ignore plugin has its own logic for what files are ignored rather than using git's.  Can it just use git's?

(By the way, I'm running .ignore 1.7.6 of 2017 Feb 23 in the latest PyCharm: PY-163.10154.50, JRE 1.8.0_112-release-408-b6x64 JetBrains s.r.o, OS Mac OS X(x86_64) v10.12.3 unknown, screens 2560x1440, 2560x1440.)

**Comment by hsz** [comment]
@adobeDan Thank you for this report. This issue was resolved mostly by Migration to the native IDE indexing #367 
It will be released with v2.0.0 as soon as other issues will be resolved.

**Comment by adobeDan** [comment]
Great, thanks for the update!  I look forward to the next release.
